.github/workflows/cicd.yaml

```diff
@@ -94,3 +94,32 @@ jobs:
       build_image: ${{ needs.build.outputs.build_image }}
       skip_integration_tests: ${{ needs.conditionals.outputs.skip_integration_tests }}
       cosign_public_key: ${{ needs.build.outputs.cosign_public_key }}
+
+  check-commit-message:
```

Comment:
nit: maybe group together the job, like all other jobs are groups together aswell ...? thinking about something like a `meta` group in which `check-commit-message` and `conditionals` reside. 

might be a lot of refactoring, since `conditionals` is used everywhere and needs to be relabeled ...? ignore if to much effort.

Reply:
Given @xopham s comment, wouldn't mix it with conditionals as it would be in group compliance, but can have a look whether that makes a reasonable group